src/infra/setup.task.ts

```diff
@@ -23,11 +23,12 @@ export const task: Task<SetupTaskArgs> = {
 		const EMAIL_ADDRESS = fromEnv('EMAIL_ADDRESS');
 		// TODO this is hacky because of `import.meta` env handling
 		const {API_SERVER_HOST_PROD, SVELTEKIT_SERVER_HOST} = await import('../lib/config.js');
+		const NODE_VERSION = 18.06;
```

Comment:
```suggestion
		const NODE_VERSION = '18';
```

The version we're on is `"18.6.0"`, don't think the number `18.06` would work as expected. I also think it's better to just pin the major version, so security updates for new instances are automatic.